packages/cairo.rb

```diff
@@ -3,23 +3,10 @@
 class Cairo < Package
   description 'Cairo is a 2D graphics library with support for multiple output devices.'
   homepage 'https://www.cairographics.org'
-  version '1.15.10-0'
-  source_url 'https://fossies.org/linux/misc/cairo-1.15.10.tar.xz'
+  version '1.15.10'
+  source_url 'http://cairographics.org/snapshots/cairo-1.15.10.tar.xz'
```

Comment:
Is the original download faster?  Also, if you make a change like this, the version should be bumped forward, not backward.
`version '1.15.10-1`